src/app.py

```diff
@@ -221,9 +228,7 @@ def app_info(app, full=False, upgradable=False):
     ret["is_webapp"] = "domain" in settings and "path" in settings
 
     if ret["is_webapp"]:
-        ret["is_default"] = (
-            domain_config_get(settings["domain"], "feature.app.default_app") == app
-        )
+        ret["is_default"] = settings.get("default_app", "_none")
```

Comment:
Hmm I am once again suspicious with this ... at some point I realized that it was obviously legit but now I am once again confused .. `settings` is supposed to contain the settings of the app, not the settings of the domain so ugh why should it contain the `default_app` key x_X